Event/EventsClassification.gd

```diff
@@ -0,0 +1,16 @@
+extends Node
+
+enum EventsClassification { EVENT_MOB, EVENT_RANDOM, EVENT_SHOP, EVENT_HEAL }
```

Comment:
the problem with an enum is that it doesn't hold classes, under the hood it's int.
The advantage of having a list of Event is to be able to directly instantiate the correct scene by checking the Array, without having to make a correspondence between `EVENT_MOB` (in the enum) and `Event_Mob` (the class)
So the function to create an event would become:
```GDScript
func createEvent(index: int) -> Event:
    return EventsClassification[index].new("res://Event/EventTest.tscn")
```

Reply:
Gotcha! I'll remove this and just stick with the Array then. That createEvent function was mainly used for testing anyways.